CommunityFridgeMapApi/dependencies/python/storage.py

```diff
@@ -0,0 +1,85 @@
+import os
+import uuid
+from urllib.parse import urlparse, urlunparse
+import boto3
+import botocore
+from botocore.config import Config
+
+def get_s3_client(env=os.getenv("Environment")):
+    endpoint_url="http://localstack:4566/" if env == "local" else None
+    config = Config(
+        signature_version=botocore.UNSIGNED, # Do not include signatures in s3 presigned-urls.
+    )
+
+    return boto3.client(
+        "s3",
+        config=config,
+        endpoint_url=endpoint_url,
+    )
+
+def translate_s3_url_for_client(url: str, env=os.getenv("Environment")) -> str:
```

Comment:
Missing unit test for this function

Reply:
Good catch :+1: 
Added a couple tests in 12cda1e53e8df5bd0428a75fc92b29bec1ee6900